dist/stat.py

```diff
@@ -36,6 +37,7 @@ def print_struct(title, name, base, stats):
             'connections', 'connection', 1000, connection_stats)
         print_struct('data sources', 'dsrc', 2000, dsrc_stats)
         print_struct('join cursors', 'join', 3000, join_stats)
+        print_struct('session_ops', 'session_op', 4000, session_op_stats)
```

Comment:
It will be better if we make description as 'session operations' instead of 'session_ops'